### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by vedi**

### CODE REVIEW

File: social-providers/ios-profile-facebook/SoomlaiOSProfileFacebook/SoomlaFacebook.m [hunk 1]
```diff
@@ -133,7 +133,7 @@ - (void)login:(loginSuccess)success fail:(loginFail)fail cancel:(loginCancel)can
  */
 - (void)getUserProfile:(userProfileSuccess)success fail:(userProfileFail)fail {
     LogDebug(TAG, @"Getting user profile");
-    [self checkPermissions: @[@"public_profile", @"user_birthday"] withWrite:NO
+    [self checkPermissions: @[@"public_profile", @"user_birthday", @"user_location", @"user_likes"] withWrite:NO
```
Comment (borisspektor) [inline social-providers/ios-profile-facebook/SoomlaiOSProfileFacebook/SoomlaFacebook.m]: @vedi why did we have to add permissions here?

Comment (vedi) [inline social-providers/ios-profile-facebook/SoomlaiOSProfileFacebook/SoomlaFacebook.m]: @borisspektor, we're just aligning implementation between platforms.

Comment (borisspektor) [inline social-providers/ios-profile-facebook/SoomlaiOSProfileFacebook/SoomlaFacebook.m]: what platform did this come from?

*Boris Spektor*
Software Developer
*M* +972.52.82.32.454
*E* boris@soom.la <refael@soom.la>
[image: Google Products] <http://soom.la/>
soom.la

Comment (EugeneButusov) [inline social-providers/ios-profile-facebook/SoomlaiOSProfileFacebook/SoomlaFacebook.m]: @borisspektor, `UserProfile` class has `location`, `language` fields, which can be read only if `user_location` (for `location`) and `user_likes` (for `language`) permissions was granted.

**Comment by vedi** [comment]
@refaelos, @borisspektor, it's ready to merge

**Pull request merged**